src/modules/addPlace/addPlace.view.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import { Flex, InputItem, List, Button, WhiteSpace, WingBlank, TextareaItem, NoticeBar } from 'antd-mobile';
+import { observable, reaction, observe } from 'mobx';
+import { inject, observer } from 'mobx-react';
+import { Bind } from 'lodash-decorators';
+import DeviceService from '../../services/DeviceService';
+import { Select } from 'antd';
```

Comment:
Try to not mix external imports with internal imports.

For readability, external imports should be first.